pkg/server/backuprestoreserver.go

```diff
@@ -194,7 +193,7 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 	defer handler.Stop()
 
 	// Promotes member if it is a learner
-	if restoreOpts.OriginalClusterSize > 1 {
+	/*if restoreOpts.OriginalClusterSize > 1 {
```

Comment:
With this block commented out, where are you promoting the member? I might've missed something

Reply:
I really need your help here. I don't understand why these lines are here. We are anyway promoting members in L310

Reply:
The `PromoteMember` in L310 is mainly targeted at cases when there is single member restoration. 
In that case, etcd-br will not restart, so these lines will not be executed, hence it's added there.

For new members, will `le.CurrentState == StateFollower` turn out to be true? If that is the case, then we can consider removing this block and let the `leaderelector` handle promotion, however we will still need the part where we update members (`UpdateMember()`) for existing clusters to have their peer URLs updated